### ISSUE TITLE
External druid data connection

### ISSUE THREAD

**Issue by hashansl** [metatron-app/metatron-discovery#1923]
Hey,

I tried to use data connection with my own druid. When I tried to create a database connection it gives this error ! ( Mysql data connection works perfectly ).
here our metatron-discovery hosted at ip like 192.172.8.205 and the druid hosted at 192.171.8.251 . When telnet the driud's port as "telnet 192.172.8.251 8081" in the terminal shows that it is connected and after trying to connect via metatron-discovery GUI using appropriate parameters of druid port and url , there was a invalid connection error. 
 
![Screenshot from 2019-04-23 01-36-00](https://user-images.githubusercontent.com/45277530/56558861-fce1ae00-656d-11e9-9a91-304185509cea.png)
  
please give me a solution for this,
Thank you.

**Comment by alchan-lee** [comment]
Hi, @hashansl 
You need to connect with broker's port.
Please try this (default is 8082).

**Comment by hashansl** [comment]
Hi @alchan-lee
I've tested the solution with the broker port number . But the again I've got the same error . The telnet parameters for "telnet 192.172.8.251 8082" shows connected . If we connected to locally hosted druid as "localhost" and port as "8082" the connection is valid and accepted. 

I wanted to resolve this .

**Comment by alchan-lee** [comment]
Sorry for the late reply.

I thought It is a network issue so I have been trying to reproduce it, but it is difficult to make the same as your status.

But I noticed your comment 
> I tried to use data connection with my own druid. 

Is it right that "my own druid" means "Apache druid (not our druid binary)"?

**Comment by hashansl** [comment]
@alchan-lee Yes. It's not metatron custom druid. I have tried it with the latest druid version.( 0.13.0)

**Comment by kyungtaak** [comment]
@hashansl I'm sorry for the late reply.
I did some simple tests with apache druid 0.14 version first.
Set the connection (jdbc:avatica:remote:url=http://localhost:8082/druid/v2/sql/avatica/) as shown below,
![image](https://user-images.githubusercontent.com/822255/57577024-cd58fe00-74a8-11e9-86b0-c7a899ceb798.png)
(reference link : http://druid.io/docs/latest/querying/sql.html)

And I tried it on the workbench as shown below.
![image](https://user-images.githubusercontent.com/822255/57577027-e6fa4580-74a8-11e9-9241-d7db57b4dea7.png)

Although apache druid I tested is a 0.14 version, the 0.13 version will do this as well. Would you like to try it?
